staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_discovery.go

```diff
@@ -172,11 +186,17 @@ var _ DiscoveryAggregationController = &discoveryManager{}
 func NewDiscoveryManager(
 	target discoveryendpoint.ResourceManager,
 ) DiscoveryAggregationController {
+	discoveryScheme := runtime.NewScheme()
+	utilruntime.Must(apidiscoveryv2scheme.AddToScheme(discoveryScheme))
+	utilruntime.Must(apidiscoveryv2beta1scheme.AddToScheme(discoveryScheme))
+	codecs := serializer.NewCodecFactory(discoveryScheme)
```

Comment:
And registering here I suppose.  I didn't expect that.  Yeah, let's avoid adding these cross conversions generally.  Having non-auto-converting schemes is something I really like about the library shape after going to staging repos.

Reply:
updated